src/components/NewClusterModal.js

```diff
@@ -280,7 +280,8 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
             name: currentPersistentDiskDetails.name
           } : {
             name: Utils.generatePersistentDiskName(),
-            size: newPersistentDisk.size
+            size: newPersistentDisk.size,
+            saturnApplication: 'runtime'
```

Comment:
Should this be `labels: { saturnApplication: 'runtime' }`?

Also TOL: I think it's fine to add this, but since there have already been PDs created without it, it will be difficult to use this value by itself to make decisions, unless we back-fill the data on the server side.

Reply:
Yeah… I thought about adding this and kept going back and forth. Let's not have it, and that will let us act under the assumption that unlabeled ones are with runtime. (I hate that sort of un-clear assumption, but I don't know if there's another way)